NftVotePlugin/accounts.ts

```diff
@@ -45,3 +57,53 @@ export const getNftVoteRecordProgramAddress = async (
     nftVoteRecordBump,
   }
 }
+
+export const getNftActionTicketsForVoter = async (
+  client: NftVoterClient,
+  registrar: PublicKey,
+  voter: PublicKey
+) => {
+  const nftActionTicketsFiltered = ((await (client.program as Program<NftVoterV2>).account.nftActionTicket.all(
+    [
+      {
+        memcmp: {
+          offset: 8,
+          bytes: registrar.toBase58(),
+        },
+      },
+      {
+        memcmp: {
+          offset: 40,
+          bytes: voter.toBase58(),
+        },
+      },
+    ]
+  )) as unknown) as NftActionTicket[]
+  return nftActionTicketsFiltered
+}
+
+export const getNftActionTicketProgramAddress = async (
```

Comment:
i thought anchor had stuff for this? 
note: i don't actually use the official anchor sdk, I always used anchor-client-gen, so i really dont know.

not a big deal either way